packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -76,26 +73,27 @@ export default function ProfileMenu({
       },
     ];
 
-    if (!isIOSNative()) {
-      list.push({
-        title: isPlus ? 'Manage plus' : plusCta,
-        buttonProps: {
-          tag: 'a',
-          icon: <DevPlusIcon />,
-          href: isPlus ? managePlusUrl : plusUrl,
-          className: isPlus ? undefined : 'text-action-plus-default',
-          target: isPlus ? '_blank' : undefined,
-          onClick: () => {
-            logSubscriptionEvent({
-              event_name: isPlus
-                ? LogEvent.ManageSubscription
-                : LogEvent.UpgradeSubscription,
-              target_id: TargetId.ProfileDropdown,
-            });
-          },
+    list.push({
+      title: isPlus ? 'Manage plus' : plusCta,
+      buttonProps: {
+        tag: 'a',
+        icon: <DevPlusIcon />,
+        href: plusHref,
+        className: isPlus ? undefined : 'text-action-plus-default',
+        target:
+          isPlus && plusProvider === SubscriptionProvider.Paddle
+            ? '_blank'
+            : undefined,
```

Comment:
Should we have some sort of URL also when managing iOS subscription?

Reply:
No, it sends a request to the App to show a modal on top of the window

Reply:
Wait, this is desktop?

Reply:
Ah, it's `_target`. So on desktop when you have iOS subscription, we redirect you to `/account/subscription`

Reply:
Ah, yes. Mislooked also, thought it was about the URL. I am not seeing things clearly, must be the glasses 😆

Reply:
Did they swap your left and right eye as well? 😂